src/status.js

```diff
@@ -0,0 +1,8 @@
+export default function checkBoxEventListener(parentCheckBoxContainer, tasksArray) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.